src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,86 @@
+import { Person } from '../Person/Person';
+import { SearchLink } from '../SearchLink';
+import { useSearchParams } from 'react-router-dom';
+import { getVisiblePeople } from '../../utils/getVisiblePeople';
+import classNames from 'classnames';
+import { usePeople } from '../../hooks/usePeople';
+
+const COLUMNS = ['Name', 'Sex', 'Born', 'Died', 'Mother', 'Father'];
+
+export const PeopleTable = () => {
+  const { people } = usePeople();
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+
+  const visiblePeople = getVisiblePeople(
+    people,
+    searchParams.get('query'),
+    searchParams.getAll('centuries'),
+    searchParams.get('sex'),
+    sort,
+    order,
```

Comment:
```suggestion
  const sort = searchParams.get('sort');
  const order = searchParams.get('order');
  const query = searchParams.get('query');
  const centuries = searchParams.getAll('centuries');
  const sex = searchParams.get('sex');

  const visiblePeople = getVisiblePeople(
    people,
    { query, centuries, sex, sort, order }
```
When you have too many parameters it's better to pass an object to avoid mistakes with order